akita/__tests__/dirty-check.spec.ts

```diff
@@ -135,6 +135,35 @@ describe('DirtyCheck', () => {
         const hasHead = dirtyCheck.hasHead();
         expect(hasHead).toBeFalsy();
       });
+
+      describe('dirtyPath',()=>{
+        const widgetsStore = new WidgetsStore({ some: { nested: { value: '' } } });
+        const widgetsQuery = new WidgetsQuery(widgetsStore);
+        const dirtyCheck = new DirtyCheckPlugin(widgetsQuery).setHead();
+        it('should not return dirty for isPathDirty', function() {
```

Comment:
The tests are not cover the whole process.

Reply:
Why? Iv'e added a test for changed value and unchanged value in both entity and store. What else should I cover?

Reply:
another cycle